docs/UserGuide.md

```diff
@@ -400,7 +401,8 @@ This will add a log to the lessons of a student. The time field of the log entry
 | **Content**        | `-content` | Yes      | Content should be a string, special characters are allowed.                                                                                                                                                                                                                               |
 | **Learning Style** | `-style`   | Yes      | Learning style should be a string, special characters are allowed.                                                                                                                                                                                                                        |
 | **Notes**          | `-notes`   | Yes      | Notes should be a string, special characters are allowed.                                                                                                                                                                                                                                 |
-
+Even though the prefixes are compulsory, their content can be left blank (except ID). <br>
+E.g. `log -id 1 -hours -content -style -notes` adds a log with blank `hours`, `content`, `style` and `notes` field.
```

Comment:
think its best to standardise this to 000001 since all the other commands uses 000001